packages/react/src/components/Link/Link.tsx

```diff
@@ -0,0 +1,31 @@
+import type { AnchorHTMLAttributes, ReactNode } from 'react';
+import React, { forwardRef } from 'react';
+import cn from 'classnames';
+
+import classes from './Link.module.css';
+
+export type LinkProps = {
+  /** The content to display inside the link. */
+  children: ReactNode;
+
+  /** Custom class name for the link. This will be appended to the design system class names. */
+  className?: string;
+
+  /** Inverts the color of the link. Use this on dark backgrounds. */
+  inverted?: boolean;
+
+  /** The URL that the link points to. This can also be an email address (starting with `mailto:`) or a phone number (staring with `tel:`). */
+  href?: string;
+} & AnchorHTMLAttributes<HTMLAnchorElement>;
+
+export const Link = forwardRef<HTMLAnchorElement, LinkProps>(
```

Comment:
hmm, I think we still need to support polyformism for this component. Libraries and frameworks such as nextjs or react-rotuer use the their on `Link` components which needs to work with our Link. 

We'll just have to implement it directly instead of using one of the typography components. Have a look at [Paragraph](https://github.com/digdir/designsystem/blob/3c987b3c264508e4f1ef6771d39cfc0ab50c6b78/packages/react/src/components/Typography/Paragraph/Paragraph.tsx#L19) for how to implement it.